modules/virt-creating-blank-disk-datavolumes.adoc

```diff
@@ -29,18 +28,17 @@ spec:
   source:
       blank: {}
   pvc:
-    # Optional: Set the storage class or omit to accept the default
-    # storageClassName: "hostpath"
+    storageClassName: "hostpath" <1>
     accessModes:
       - ReadWriteOnce
     resources:
       requests:
         storage: 500Mi
 ----
+<1> Optional: If you do not specify a storage class, the default storage class is applied.
```

Comment:
I guess the default storage class is identified somewhere close by?

Reply:
The default storage class is whatever the cluster admin has configured it to be. Since the cluster admin is usually the same person creating a blank DV, we can assume that they know what the default storage class is.